django/contrib/gis/forms/widgets.py

```diff
@@ -61,11 +61,13 @@ def get_context(self, name, value, attrs):
                         value.srid, self.map_srid, err
                     )
 
+        geom_type = str(gdal.OGRGeomType(self.attrs['geom_type']))
```

Comment:
Using `.name` would be better than `str()`:

```suggestion
        geom_type = gdal.OGRGeomType(self.attrs['geom_type']).name
```